integrations/to.etc.domui.selenium/src/main/java/to/etc/domui/webdriver/core/base/RadioButtonPO.java

```diff
@@ -0,0 +1,34 @@
+package to.etc.domui.webdriver.core.base;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import to.etc.domui.webdriver.core.WebDriverConnector;
+
+@NonNullByDefault
+public class RadioButtonPO extends BasePO {
+	private final By m_locator;
+
+	public RadioButtonPO(WebDriverConnector wd, String id) {
+		super(wd);
+		m_locator = wd.byId(id);
+	}
+
+	public void select(int order) {
```

Comment:
All methods that invoke an action are called click() so far, so why not call this click()?